core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -236,6 +243,16 @@ private FunctionResolver hour() {
     );
   }
 
+  private FunctionResolver makedate() {
+    return define(BuiltinFunctionName.MAKEDATE.getName(),
+        impl(nullMissingHandling(DateTimeFunction::exprMakeDate), DATE, DOUBLE, DOUBLE));
```

Comment:
Why DOUBLE?  These should be integers, no?  I don't know what makedate(1.5, 1.5) means.

Reply:
These are rounded...?

Reply:
MySQL accepts doubles and round them, I did the same. Yes, I was surprised too.

Reply:
That's... unexpected.  
I need to learn to expect the unexpected from SQL language spec.